index.js

```diff
@@ -153,6 +153,11 @@ module.exports = {
     }
   },
 
+  included: function(app) {
+    this._super.included(app);
+    app.import('vendor/percy-jquery.js', {type: 'test'});
```

Comment:
Need to think about this slightly more — we technically support people right now exporting PERCY_TOKEN in development mode, running `ember server` (instead of `ember test`) and navigating to /tests and getting snapshots. I don't think that's a common flow or one we care to support, but need to think slightly more about this.

Reply:
So this will break the ability to go to `localhost:whatever/tests`? Or only getting percy snapshots locally through `localhost:whatever/tests`?

Reply:
Only getting percy snapshots when running /tests locally, which I don't think anyone does.